proto/spire/plugin/agent/workloadattestor/v1/workloadattestor.proto

```diff
@@ -1,30 +1,23 @@
-/** Environment specific plugin to attest a workloads “selector”
-data. a*/
-
 syntax = "proto3";
-package spire.agent.workloadattestor;
-option go_package = "github.com/spiffe/spire/proto/spire/agent/workloadattestor";
-
-import "spire/common/common.proto";
-import "spire/common/plugin/plugin.proto";
+package spire.plugin.agent.workloadattestor.v1;
+option go_package = "github.com/spiffe/spire-plugin-sdk/proto/spire/plugin/agent/workloadattestor/v1";
 
-/** Represents the workload PID.*/
 message AttestRequest {
-    /** Workload PID */
+    // Required. The process ID of the workload to attest.
     int32 pid = 1;
 }
 
-/** Represents a list of selectors resolved for a given PID. */
 message AttestResponse {
-    /** List of selectors */
-    repeated spire.common.Selector selectors = 1;
+    // Optional. Selector values related to the attested workload. The type
+    // of the selector is inferred from the plugin name.
+    repeated string selector_values = 1;
 }
 
 service WorkloadAttestor {
-    /** Returns a list of selectors resolved for a given PID */
+    // Attests the specified workload process. If the process is not attestable
+    // (i.e. not a process the attestor knows anything about), INVALID_ARGUMENT
+    // is returned. Otherwise the attestor returns zero or more
+    // selector values, or another status if there is an error attesting the
+    // workload.
```

Comment:
What does it mean for an attestor to not know anything about a process? And in what cases should we return zero selectors? It would be good to get really clear about exactly the behavior here, and the contract between core and plugin

As it stands today, plugins will return an error if they know they should be able to attest the process but can't for some reason. If they detect that the process is not one that is applicable to them, they return zero selectors. I suspect we want to preserve that behavior in the absence of a larger motivating conversation?